src/apiConstants.ts

```diff
@@ -28,6 +28,7 @@ export const LPS_OPPFOLGINGSPLAN_MOTTAK_V1_ROOT =
   "/lps-oppfolgingsplan-mottak/api/internad/v1";
 export const SYFOPERSON_ROOT = "/syfoperson/api/v2";
 export const SYFOSMREGISTER_ROOT = "/syfosmregister/api/v2";
+export const SYFOOVERSIKTSRV_ROOT = "/syfooversiktsrv/api/v2";
```

Comment:
Som vi snakket om - litt dumt hvis denne må være bare /api/v2 også her fra modia enkeltperson, i så fall bør vi kanskje legge til noe foran i pathen i syfooversiktsrv for det nye endepunktet :thinking: